modules/twitch_streaming

```diff
@@ -0,0 +1,59 @@
+"""
+Checks if a streamer is online. [Gross Gore/grossie_gore as placeholder]
+
+Checks if a streamer is online using the Twitch Kraken API to see
+if a channel is currently streaming or not.
+
+Configuration parameters
+    cache_timeout: how often we refresh this module in seconds
+    stream_name: name of streamer(twitch.tv/<stream_name>)
+
+Format of status string placeholders
+    {stream_name}:  name of the streamer (twitch.tv/<stream_name>)
+
+@author Alex Caswell horatioesf@virginmedia.com
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+import requests
+
+
+class Py3status:
+    # available configuration parameters
+    # can be customized in i3status.conf
+    cache_timeout = 10
+    stream_name = "grossie_gore"
+    format = "{stream_name} is live!"
+    format_offline = "{stream_name} is offline."
+
+    def get_display_name(self, stream_name):
```

Comment:
You need to change this function to `_get_display_name` otherwise py3status will try and run it.

Also this result is unlikely to change so you probably don't want to keep checking every 10 seconds.